browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2213,6 +2234,32 @@ index 5439529aa796093989ebd43adaeef88c4ddffb27..4d00bd1d5e35a845f28e697adc7cb3c4
  bool AccessibilityObject::accessibilityIsIgnored() const
  {
      AXComputedObjectAttributeCache* attributeCache = nullptr;
+diff --git a/Source/WebCore/css/MediaQueryEvaluator.cpp b/Source/WebCore/css/MediaQueryEvaluator.cpp
+index 11619a9688856b4488d059273359757da3978a89..5b9a85c66cfef8112ff2e453c1e67e01f146489b 100644
+--- a/Source/WebCore/css/MediaQueryEvaluator.cpp
++++ b/Source/WebCore/css/MediaQueryEvaluator.cpp
+@@ -843,20 +843,7 @@ static bool prefersContrastEvaluate(CSSValue* value, const CSSToLengthConversion
+ 
+ static bool prefersReducedMotionEvaluate(CSSValue* value, const CSSToLengthConversionData&, Frame& frame, MediaFeaturePrefix)
+ {
+-    bool userPrefersReducedMotion = false;
```

Comment:
Just to reduce the size of our diff I'd rather keep this logic here and have a quick check of the override right before.